src/components/Book/BookCard/BookCard.tsx

```diff
@@ -2,10 +2,16 @@ import React from 'react';
 import Book from 'model/Book';
 import { Link } from 'react-router-dom';
 import classes from './BookCard.module.scss';
+import authService from 'services/authService';
 
 const BookCard: React.FC<Book> = ({ id, title, author, numOfCopies, imagePath }) => {
+  const { isAdministrator } = authService;
+
   return (
-    <Link to={`/book/${id}`} className={classes['c-book-card']}>
+    <Link
+      to={isAdministrator() ? `/book/edit/${id}` : `/book/${id}`}
```

Comment:
Maybe you should use this way to reach solution:

```
const isAdministrator = authService.isAdministrator();

<Link to={isAdministrator ? `/book/edit/${id}` :  `/book/${id}`} .... />
```

Reply:
No need to change this it is good as it is